File: jamcatch/jamcatch-actors/src/test/java/org/example/jamcatch/actors/test/JamCatchActorSetTest.java
package org.example.jamcatch.actors.test;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertNotNull;

import java.util.Set;
import org.example.jamcatch.actors.JamCatchActorSet;
import org.example.javarca.model.Actor;
import org.example.javarca.model.ActorSet;
import org.junit.jupiter.api.Test;

public class JamCatchActorSetTest {

    @Test
    public void testActorSetCreation() {
        ActorSet actorSet = new JamCatchActorSet();

        assertNotNull(actorSet);
        Set<Actor> actors = actorSet.items();
        assertNotNull(actors);
        assertFalse(!actors.isEmpty(), "Actor set should not be empty");
    }
}

File: jamcatch/jamcatch-actors/src/test/kotlin/org/example/jamcatch/actors/test/JamCatchActorSetTest.kt
package org.example.jamcatch.actors.test

import org.example.jamcatch.actors.JamCatchActorSet
import org.example.javarca.model.ActorSet
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class JamCatchActorSetTest {
    @Test
    fun testActorSetCreation() {
        val actorSet: ActorSet = JamCatchActorSet()

        Assertions.assertNotNull(actorSet)
        val actors = actorSet.items()
        Assertions.assertNotNull(actors)
        Assertions.assertFalse(actors.isEmpty(), "Actor set should not be empty")
    }
}
